CIP-XXXX/README.MD

```diff
@@ -0,0 +1,146 @@
+
+- **Key Registration:** Once derived, register these stake keys. This registration process remains consistent with current procedures, with the added context that all these new stake keys will be associated with a common payment key.
+
+- **Fund Distribution:** Distribute the funds across addresses formed by pairing the first  payment key with each of the newly created stake keys. This creates an association between all stake delegations. The details of how funds are distributed are specified by the user when creating the multi delegation transaction.
+
+Wallet software can then easily discover all stake keys and associated delegations either by sequential discovery or identifying addresses  containing the first payment credential. 
+
+### Updating the Multi-Delegation
+The dynamic nature of multi-delegation facilitates not just participation but also continuous optimization of delegations.
+
+#### Adding Pools
+To delegate to additional pools, the first step would involve deriving the necessary number of new stake keys. Then, make a transaction that:  registers the new stake keys, delegates and  distributes funds among addresses that pair the new stake keys with the first payment key.
+
+#### Removing Pools
+To remove a pool, the wallet first identifies all addresses associated with the stake key designated for removal. Then, make a transaction that
+Transfer all funds from these addresses
+withdraw all rewards from their associated reward accounts to one of the accounts that are still in delegation. If no other delegating accounts are left, revert the funds to addresses associated with the first payment key. 
+. Deregister stake keys
+
+**Note that all rewards calculated but not distributed will be lost.**
+
+#### Interoperability
+Live delegation portfolio might drift overtime due to:
+- Receiving funds to a single address 
+- Building transactions with software that doesn’t support multi-stake delegation (e.g. via dapp connector)
+
+In order to maintain a record of user’s delegation portfolio preferences , all actions pertaining to stake delegation, including funds transfers, stake key registration, and deregistration, must be executed within a single atomic transaction when performing an update. This transaction should also include the delegation portfolio metadata, using the reserved metadata label 6862 (hexadecimal 0x1ace), as outlined in [CIP-0017](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0017) preferably without any of the optional fields.
+
+Example:
+
+``` Json
+{
+  "pools": [
+    {
+      "ticker": "DARK",
+      "weight": 42
+    },
+    {
+      "id": "5f3833027fe8c8d63bc5e75960d9a22df52e41bdf62af5b689663c50",
+      "weight": 14
+    },
+    {
+      "id": "a16abb03d87b86f30bb743aad2e2504b126286fe744d3d2f6a0b4aec",
+      "weight": 37
+    },
+    {
+      "id": "9f9bdee3e053e3102815b778db5ef8d55393f7ae83b36f906f4c3a47",
+      "weight": 25
+    }
+  ]
+}
+```
+
```

Comment:
I would strongly recommend that we do not allow for the use of "ticker" or any other non-cryptographically derived identifier for any on-chain data that may be consumed by third parties. This can create an ambiguity if multiple pools are registered with the same ticker.

Furthermore, for ease of implementation and consistent interoperability between various implementations I would say that we should definitively settle on either the hex pool ID representation or the bech32 representation.

Reply:
yes, I also think it's important that we avoid putting pool "ticker" references on chain.  When I advocated for pool tickers in stake pool URIs, we hotly debated using tickers at all in the "delegation portfolio" links in both CIP-0013 and CIP-0017 and could only collectively justify using tickers when both
1. off-chain (leaving wallets to settle ambiguities); and
2. in user-facing cases: where brevity, human readability, and/or convenience are important: i.e not in the off-chain JSON spec of CIP-0017 which is written for precision and where brevity isn't important.

Reply:
Agree, it is an error, all entries should be id + weight, although the CIP just makes the suggestion to not use optional fields (such as ticker), we could make it mandatory to have this shape, id (hex pool ID) + weight

Reply:
done here https://github.com/cardano-foundation/CIPs/pull/628/commits/cdafdd40bf23e4eb200bff240c1af2d6c9974eb2